redfish_guide/topics/using-redfish-physical-infra.adoc

```diff
@@ -195,3 +195,65 @@ the selected image and customization template, has set the servers to boot once
+physical systems. The Firmware Update functionality consists of the following:
+
+* *Firmware Registry integration*. Firmware Registry is an external service that contains
+  a list of binaries that are the payload in the firmware update process. The registry
+  integration provides the inventory of the available firmware binaries tagged by
+  manufacturer and system model. Only compatible binaries are offered to install on a
+  specific server.
+* *Firmware Update request*. Firmware Update process is implemented in a form of
+  {product-title_short} request which can be initialized either from physical server details
+  page or list page. The latter allows one to trigger firmware update process on a number
+  of servers in one go.
+
+NOTE: This guide assumes we have firmware registry already set up and registered with
+{product-title_short}. Moreover, this guide assumes registry contains at least one
+binary compatible with the physical server we are targeting for update.
+
+We start off by visiting the physical servers list in *Compute -> Physical
+Infrastructure -> Servers*. On the server list, we check one or more checkboxes
+to select the servers to be targeted for firmware update. This enables the
+*Lifecycle* menu.
+
+image::firmware_update/center_menu.png[alt="Lifecycle menu for Firmware Update of Selected Physical Servers",550,align="center"]
+
+Click the *Lifecycle -> Update Firmware of Physical Servers*. This opens the
+dialog for filling in the request.
+
+image::firmware_update/popup.png[alt="Update Physical Server's Firmware dialog",350,align="center"]
+
+NOTE: Same dialog can be accessed from withing physical server's details page.
+
+The dialog informs us on the number of servers selected for this request. It
+presents the following fields that we need to fill:
+
+* *Firmware Binary*: selection of the firmware binary to install on selected physical servers
+
+image::firmware_update/popup_filled.png[alt="Update Physical Server's Firmware dialog (filled)",350,align="center"]
+
+NOTE: Only compatible firmware binaries are available to choose from based on your current
+selection of physical servers. If you don't have any firmware binaries
+listed this means that either you don't have Firmware Registry set up correctly (visit
+*Compute -> Infrastructure -> Firmware Registry* to set it up) or that there are simply no
+compatible firmware binaries available for selected physical server.
+
+
+Select desired firmware binary from the list and click *Apply Firmware*.
+
+image::firmware_update/request_created.png[alt="Firmware Update request created",350,align="center"]
+
+Firmware Update request gets executed without manual approval (it gets auto-approved). You can
+track its progress by visiting *Services -> Requests* and look for the request of type
+Physical Server Firmware Update.
+
+image::firmware_update/request_list_view.png[alt="Firmware Update request completed (list view)",500,align="center"]

```

Comment:
This image is difficult to read too.

Reply:
FYI the size, the content and the readability of the image matches exactly what we have for "Provisioning request provisioned" in the previous chapter. But I agree, we can enlarge the image to 800 and it's much better readable right away.

Reply:
You're right, I need to make those bigger, too.